manager/src/main/java/com/redhat/service/bridge/manager/models/Processor.java

```diff
@@ -82,6 +81,9 @@
     @OneToMany(mappedBy = "processor", fetch = FetchType.LAZY, orphanRemoval = true, cascade = CascadeType.ALL)
     private Set<Filter> filters;
 
+    @Column(name = "transformationTemplate")
```

Comment:
Can you make this `transformation_template` so that our column naming is consistent.

Reply:
Yep good spot